### PR TITLE
Log identifier for lambda containers

### PR DESCRIPTION
**Pull request by bencodeorg**
Adds a persistent identifier for lambda containers to our logging.

Jessie remembered reading that the request handler persists across the lifetime of a lambda container (~~we can't find the source of this information~~ check out initialization code section [here for more information](https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html)), so that is where we implemented this change.

We did try to prove it out by the following though:
- repeated runs of the same program on a dev deploy return the same UUID (as verified via Postman)
- if one program is executing (ie, by putting a wait in such that the program does not complete its execution), executing another program returns a different UUID

This change should log the container ID any time we log to cloudwatch.

### CODE REVIEW

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java [hunk 1]
```diff
@@ -34,6 +36,7 @@
   private static final int CHECK_THREAD_INTERVAL_MS = 500;
   private static final int TIMEOUT_WARNING_MS = 20000;
   private static final int TIMEOUT_CLEANUP_BUFFER_MS = 5000;
+  private static String LAMBDA_ID = "";
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java:39]: nit: initialize to `null` and check if null before assigning?

**Review by sanchitmalhotra126 — APPROVED** [review]
Looks good! One small nit

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java (hunk 1 above)
Comment (molly-moen) [reply org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java:39]: per the [docs](https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html)(search for "initialization code" on that page), the way to docs recommend initializing a static variable would be:
```
private static String LAMBDA_ID = UUID.randomUUID().toString();
```
This way works too, but doing it above the handler is a bit cleaner.

Comment (molly-moen) [reply org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java:39]: looks like alternatively we could create a constructor and initialize things there as well.

Comment (bencodeorg) [reply org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java]: hah well that is probably where jessie remembered this from :). Simplified the logic, thanks for the info

**Review by molly-moen — APPROVED** [review]
🎉

**Pull request merged**